docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -0,0 +1,61 @@
+
+## Process for starting a new translation
+
+The first step for starting a new translation is to check what exists. So far, there are repositories for the following languages:
+
+- [Spanish](https://github.com/gatsbyjs/gatsby-es)
+- [Russian](https://github.com/gatsbyjs/gatsby-ru)
+
+> Note: Once a new translation repository is created, feel free to add it here in a PR!
+
+### Finding 2-3 codeowners
+
+For a new translation, open an issue with information about your intended language. If you already have co-contributors to act as fellow [code owners](https://help.github.com/en/articles/about-code-owners) and provide checks and balances for PR reviews and quality assurance, that would be very helpful! Otherwise, you can check out the [RFC](https://github.com/gatsbyjs/rfcs/pull/42) for interested participants or [get in touch with us](/contributing/how-to-contribute/#not-sure-how-to-start-contributing) at Gatsby for help in building your translation team.
+
+## Contributing to a translation
+
+Once a language repository is created and someone on the Gatsby team has assigned codeowners, contributions can begin. It is up to the discretion of the contributor how exactly they want to work, but it's recommended to limit the scope of PRs to 1-3 docs at a time to aid with code reviewing.
+
+Maintainers of a translation are expected to review translations for quality, friendly and encouraging tone of voice, and technical accuracy. These qualities should be considered in every pull request regardless of who it comes from; contributions should be encouraged from anyone in the community, not just maintainers.
+
+If disputes arise, having a conversation on Discord or over a GitHub issue should be the start. If all attempts have been made and something can not be resolved on your own, you can contact the Gatsby team at [conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com) with details about the situation.
+
+### Code review process
+
+    - Invite comments for a code review process (Google Translate, etc.)
+
+### Templates for responses on PRs
+
+    - PR templates for responses (e.g. non-meaningful changes, etc.)
+
+## Which decisions are flexible? Which are firm guidelines?
+
+## Language-specific maintainer channels
+
+Each translation group may want to have your own communication channels for maintainers and community members to ask questions and coordinate the project. Discord is the most recommended option but there are alternatives, and maintainers can ultimately choose their own communication channel(s). Here are some recommendations:
+
+- [Discord](https://gatsby.dev/discord)
+- Wechat
+- Whatsapp
+- GitHub groups
```

Comment:
@tesseralis @marcysutton this is not a good idea actually. GitHub teams/groups are very convenient to automatically request a review, and to inform all teams that are engaged only in translations, something important.

I also remind you that you can control notifications from groups as you wish.

![image](https://user-images.githubusercontent.com/4408379/66724015-331b2e80-ee29-11e9-80fa-2120a4926b3e.png)

* * *

I beg you to reconsider this decision, or to conduct a survey to make the determination, because you come from your experience as the translations curator, and therefore you can’t know how GH groups are helped to maintainers.

Reply:
:warning: Very urgent!

Explain to me how a completely stranger could make changes directly to the `master` branch without merge pull request?!

Please take a look here: https://github.com/gatsbyjs/gatsby-ru/commit/b6301dce6f229a9e71a8000b8eaa177d37a0c81c#commitcomment-35474697

I'm as a code owner for some reason can’t do this, but some person who has nothing to do with Russian translation could do it! This is a machine translation, if that!

Please stop ignoring me, I am writing about important things that help the our community!

Reply:
@lex111 The notification feature is available with codeowners. Codeowners are automatically requested for review in a PR. The reason you weren't for the Russian translation was because you weren't already a member of the Gatsby organization. You should be one now, and should have the proper permissions to review and make changes.

The issue with the spam contributor has also been resolved. Thank you for bringing this up.

Reply:
@tesseralis yes, I did not know this, and wrote about it many times, but no one answered, but it was good that it was decided :+1: